### PR TITLE
Live observations on homepage

### PR DESCRIPTION
**Pull request by dereklieu**
Needs a bit more work. I realized that the id's that get included from the `listObservations` endpoint aren't the same as the internal osm db id's. This should be an easy adjustment though.

### CODE REVIEW

**Comment by dereklieu** [comment]
@noffle this uses the timestamp index code to get the most recent observations, then fetches the requisite observation data using the timestamp version id from the redux store. Would love to get a quick look from you at those bits, especially [db](https://github.com/osmlab/field-data-coordinator/pull/76/files#diff-c8c8bcfbad4656e860b3b9bab4ecd485) and [selectors](https://github.com/osmlab/field-data-coordinator/pull/76/files#diff-fd1b991277e9da9983ef71403d7f3f91).

File: src/lib/db.js [hunk 1]
```diff
@@ -186,6 +165,34 @@ function createObservation (feature, nodeId, cb) {
   })
 }
 
+function getObservationById (id, cb) {
+  observationsDb.get(id, (err, doc) => {
```
Comment (noffle) [inline src/lib/db.js]: This can be simplified down to

```js
   observationsDb.get(id, cb)
```

Comment (noffle) [inline src/lib/db.js]: Bonus: it won't explode if `cb` is `null`.

File: src/lib/db.js [hunk 2]
```diff
@@ -186,6 +165,34 @@ function createObservation (feature, nodeId, cb) {
...
+  observationsTimestampIndex.ready(() => done(null, observationsTimestampIndex.getDocumentStream(opts)))
+}
+
+function listSequentialObservations (cb) {
+  const ids = []
+  getObservationTimestampStream((err, stream) => {
+    if (err) cb(err)
+    stream.on('data', d => ids.push(d))
+    stream.on('end', () => cb(null, ids))
+    stream.on('error', error => cb(error))
```
Comment (noffle) [inline src/lib/db.js]: I don't think Readable streams guarantee that `'error'` will only be emitted once. I'd feel safer if we wrapped `cb` in a [`once`](https://www.npmjs.com/package/once).

File: src/lib/db.js [hunk 3]
```diff
@@ -207,18 +216,29 @@ function listObservations (cb) {
...
+function observationToFeature ({ version_id, value }, id) {
   var feature = {
     id,
     type: 'Feature',
     geometry: null,
-    properties: Object.assign({ id }, obs.tags)
+    properties: Object.assign({ id, _version_id: version_id }, value.tags)
   }
-  if (obs.lon && obs.lat) {
+  if (value.lon && value.lat) {
```
Comment (noffle) [inline src/lib/db.js]: `&& value.type === 'observation'`

Comment (noffle) [inline src/lib/db.js]: ^ This could also be an assertion up top.

Comment (dereklieu) [inline src/lib/db.js]: This should be covered in the [parent function](https://github.com/osmlab/field-data-coordinator/blob/f18de01f9d1a83ec618943d662a586547fad4810/src/lib/db.js#L207). `observationToFeature` is also not used anywhere else, and it feels weird to return `null` or something like that at this stage.

**Pull request merged**